susemanager-utils/susemanager-sls/susemanager-sls.changes

```diff
@@ -1,3 +1,4 @@
+- Logrotate salt SSH logs
```

Comment:
Minor nitpick on the message here, maybe something like:

```suggestion
- Enable Salt logs for SSH minions and activate logrotate
```

The other message sounds a bit poor.

Reply:
@meaksh I am not opposed but susemanager-sls does not have anything to do with enabling the logs. That's done by `spacewalk-setup`

Reply:
Fixed.